frontend/src/views/Search/Search.tsx

```diff
@@ -24,7 +25,7 @@ export const Search = () => {
     setSearchState({ status: "LOADING" });
     try {
       const results = await search(data);
-      setSearchState({ status: "SUCCESS", results });
+      setSearchState({ status: "SUCCESS", results, phrase: data.phrase });
```

Comment:
Nie wiem dlaczego, ale tutaj results (zwrócone z backendu jako tablica) zostają rozpakowane z tablicy do obiektu (być może jest to wina tego, że jest tylko jeden element tablicy, ale i tak dobrze by to było obłużyć), co powoduje później error w `SearchResultsList`, gdy zachodzi próba odpalenia metody `state.results.map()`

Reply:
To znaczy aktualnie results to jest tablica tablic obiektów, w której każdy element zawiera wyniki z danego mikroserwisu. Na razie jest tylko jeden mikroserwis, więc ta tablica najbardziej zewnętrzna zawsze ma tylko 1 element. Docelowo to będzie tablica obiektów, więc na razie nie trzeba się o to martwić.